lib/ansible/modules/database/aerospike/aerospike_migrations.py

```diff
@@ -0,0 +1,562 @@
+#!/usr/bin/python
+"""short_description: Check or wait for migrations between nodes"""
+
+# Copyright: (c) 2018, Albert Autin
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aerospike_migrations
+short_description: Check or wait for migrations between nodes
+description:
+    - This can be used to check for migrations in a cluster.
+    - This makes it easy to do a rolling upgrade/update on Aerospike nodes.
+    - If waiting for migrations is not desired, simply just poll until
+    - port 3000 if available or asinfo -v status returns ok
+version_added: 2.8
+author: "Albert Autin (@Alb0t)"
+options:
+    host:
+        description:
+            - Which host do we use as seed for info connection
+        required: False
+        type: str
+        default: localhost
+    port:
+        description:
+            - Which port to connect to Aerospike on (service port)
+        required: False
+        type: int
+        default: 3000
+    connect_timeout:
+        description:
+            - How long to try to connect to node before giving up (ms)
+        required: False
+        type: int
+        default: 1000
+    consecutive_good_checks:
+        description:
+            - How many times should the cluster report "no migrations"
+            - consecutively before returning OK back to ansible?
```

Comment:
```suggestion
              consecutively before returning OK back to ansible?
```
Same as above. Every item in this list will be rendered as a new paragraph.

Reply:
(Happens several times more. I guess you will find them yourself :) )